guides/recipes/gateway/README.md

```diff
@@ -40,11 +44,30 @@ The following recipes are available for deploying the gateway and httproute.
     </TabItem>
 </Tabs>
 
+### Install HTTPRoute Only
```

Comment:
We should consider migrating other guides to use the gateway recipe entirely instead of doing this special case.